lib/shortcuts/lambda.js

```diff
@@ -186,6 +186,27 @@ class Lambda {
           ],
           MetricName: 'Errors'
         }
+      },
+
+      [`${LogicalName}LogPolicy`]: {
+        Type: 'AWS::IAM::Policy',
+        Condition,
+        Properties: {
+          PolicyName: 'lambda-log-access',
+          Roles: (RoleArn) ? RoleArn : { 'Ref': `${LogicalName}Role` },
```

Comment:
Nervous here because it looks like this might mismatch role names (Ref value for a role) with role arns (per the variable name). 

Do you know whether this CloudFormation property accepts name or arn or either?

Reply:
yeah that makes me a little nervous too - I need to test it. The [docs note](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html#cfn-iam-policy-roles) that the name of the role should be used, but I know sometimes you can mix and match them alright

Reply:
So it turns out you do need to provide just the role name with these policies - you'll get an error if you provide the full ARN. But the Lambda Function resource requires the full ARN 😭 

As a result, I added [some logic](https://github.com/mapbox/cloudfriend/pull/113/files#diff-134b26139ee3cef21ec6f95e2cc95e970fb15e76fb675e5600cc2bf8f80a6c72R124-R130) to handle this. It's now working in my test repository.